mkosi.credentials/userdb.user.testuser

```diff
@@ -0,0 +1,16 @@
+{
+    "userName": "testuser",
+    "homeDirectory": "/home/testuser",
+    "uid": 4711,
+    "gid": 4711,
```

Comment:
and maybe drop this too, to ensure the gid fallback logic path also works correctly